components/Editor.vue

```diff
@@ -186,6 +187,22 @@ const getJournalUpdatedDate = () => {
         >
           Delete
         </UButton>
+        <!-- Information button -->
+        <div
```

Comment:
@mayramtv , looks cool :) 
Just one suggestion: let's use [Tooltip](https://ui.nuxtlabs.com/overlays/tooltip) or [Popover](https://ui.nuxtlabs.com/overlays/popover) component (popover can be available on [hover](https://ui.nuxtlabs.com/overlays/popover#mode) ).
In this case, information text will overlay content and editor or Delete button won't jump

Reply:
@lenabaidakova Thank you for the tip. That was what I was trying to aim for, but I didn't know how. I'll change it today.

Reply:
@lenabaidakova I updated the button with the popover. Thank you for reviewing the PR